flintrock/flintrock.py

```diff
@@ -173,10 +176,14 @@ def cli(cli_context, config, provider):
               help="Spark release version to install.")
 @click.option('--spark-git-commit',
               help="Git commit hash to build Spark from. "
-                   "--spark-version and --spark-git-commit are mutually exclusive.")
+                   "Can be 'latest' to build Spark from the latest available commit on the "
+                   "default branch. "
+                   "--spark-version and --spark-git-commit are mutually exclusive.",
+              default="latest",
```

Comment:
FYI: I am going to remove this default setting because it breaks users who have configured `--spark-version` in their config file. I may have instructed you to set this default; apologies.